themes/default/content/templates/container-service/aws/index.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Container Service on AWS
+meta_desc: The Containers template makes it easy to deploy a containerized application on AWS with Pulumi and Amazon ECS.
```

Comment:
I would like @cnunciato to weigh in here. Is the important thing here deploying the containerized app or is the important thing here that you are deploying the Container Service which is running a container app?

Reply:
It's the second thing. Many users will be interested in deploying containerized applications, and you can extend this template to deploy an application, but I think if we center on "service" and then discuss applications we'll be good.